check.py

```diff
@@ -33,14 +34,16 @@ def get_grads(variables):
 
 def check_forward(variables, with_cuda, verbose):
     baseline_values = python.lltm_baseline.LLTMFunction.apply(*variables)
-    cpp_values = cpp.lltm.LLTMFunction.apply(*variables)
+    cpp_variables = [v.cpu() for v in variables]
+    cpp_values = torch.ops.myops.lltm(*cpp_variables)
```

Comment:
It looks like it's something you already call from  LLTMFunction in `cpp/lltm.py`, so why not keep the `LLTMFunction` object here as it was before and place the backend dependent code in evry module (cpp and cuda) ?

Reply:
Since this has been migrated into an operator that is registered via dispatcher API, python implementation of `LLTMFunction` no longer exists (see `cpp/lltm.cpp:115`).

If I understand the intent of the code correctly, this use of `LLTMFunction` was a way to directly call the C++ `forward` implementation via python, but now we are using the dispatcher API, and can call the `forward` op directly.  (I think ideally the baseline solution would be updated to match this pattern too.)

Reply:
C++ LLTMFunction impl is primarily used for re-dispatching from autograd op, as per:  https://pytorch.org/tutorials/advanced/dispatcher.html#adding-autograd-support